dev/PushNotifications/PushNotificationChannel.h

```diff
@@ -29,6 +30,17 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         // IWpnForegroundSink
         HRESULT __stdcall InvokeAll(_In_ ULONG length, _In_ byte* payload, _Out_ BOOL* foregroundHandled) noexcept;
 
+        //INotificationHandler
+        HRESULT __stdcall OnRawNotificationReceived(unsigned int payloadLength, _In_ byte* payload, _In_ HSTRING /*correlationVector */) noexcept
```

Comment:
We probably should file a bug here to use the correlation vector in our telemetry?

Reply:
Yeah I created one: https://github.com/microsoft/WindowsAppSDK/issues/1464